# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse/src/unstable.rs
# Commit: 00d82acfaa2f
# Full Hash: 00d82acfaa2f56095e5033a6905427d43b95eccb
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r 4ba88410cdf6 -r 00d82acfaa2f third_party/rust/mp4parse/src/unstable.rs
--- a/third_party/rust/mp4parse/src/unstable.rs	Fri Jan 27 18:35:14 2023 +0000
+++ b/third_party/rust/mp4parse/src/unstable.rs	Fri Jan 27 18:35:14 2023 +0000
@@ -106,7 +106,7 @@
 /// sample data offset (start and end), composition time in microseconds
 /// (start and end) and whether it is a sync sample
 #[repr(C)]
-#[derive(Default, Debug, PartialEq)]
+#[derive(Default, Debug, PartialEq, Eq)]
 pub struct Indice {
     /// The byte offset in the file where the indexed sample begins.
     pub start_offset: CheckedInteger<u64>,
@@ -489,7 +489,7 @@
     })
 }
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Eq)]
 pub struct Microseconds<T>(pub T);
 
 /// Convert `time` in media's global (mvhd) timescale to microseconds,